src/main/java/org/jenkinsci/plugins/oic/OicServerManualConfiguration.java

```diff
@@ -155,7 +155,8 @@ public OIDCProviderMetadata toProviderMetadata() {
             // rather we just say "I support anything, and let the check for the specific ones fail and fall through
             ArrayList<JWSAlgorithm> allAlgorthms = new ArrayList<>();
             allAlgorthms.addAll(JWSAlgorithm.Family.HMAC_SHA);
-            allAlgorthms.addAll(JWSAlgorithm.Family.SIGNATURE);
+            allAlgorthms.addAll(JWSAlgorithm.Family.RSA);
+            allAlgorthms.addAll(JWSAlgorithm.Family.EC);
```

Comment:
Shouldn't we be filtering here `allAlgorithms` here?

Reply:
these are the compliant algorithms, thats why haven't put filter here.

Reply:
wrong filter